source/PPApp.cpp

```diff
@@ -84,6 +84,10 @@ void PanicPainterApp::update(float timestep) {
                 // TODO: Now it is screen.
                 _gameplay.loadLevel("gameplay");
             }
+            /*
+            else if (_gameplay.isComplete()) {
+                _currentScene = MENU_SCENE;
+            }*/ 
             break;
```

Comment:
Where logic can go determining if a level is complete & when the screen should be transitioned (likely need to set timer to wait long enough for player to read message)